.github/workflows/ci.yml

```diff
@@ -44,15 +44,15 @@ jobs:
         run: |
           cargo check --release --locked --features=runtime-benchmarks,try-runtime
 
-  check-ismp:
-    needs: lint
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: "./.github/actions/init"
-      - name: Check Build with ISMP
-        run: |
-          cargo check --release --locked --features=ismp,runtime-benchmarks,try-runtime
+#  check-ismp:
```

Comment:
Since this has been merged into `main`, it's worth leaving clear TODOs for the commented-out code.